master/go.mod

```diff
@@ -21,7 +21,7 @@ require (
 	github.com/go-pg/pg/v10 v10.10.6
 	github.com/golang-jwt/jwt v3.2.2+incompatible // indirect
 	github.com/golang/protobuf v1.5.2
-	github.com/google/go-cmp v0.5.7 // indirect
+	github.com/google/go-cmp v0.5.8 // indirect
```

Comment:
i'm not revert to this version. locally i see this error: 
go install github.com/google/go-cmp@v0.5.7
go: github.com/google/go-cmp@v0.5.7: module github.com/google/go-cmp@v0.5.7 found, but does not contain package github.com/google/go-cmp